servicetalk-examples/grpc/helloworld-maven/pom.xml

```diff
@@ -0,0 +1,78 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.servicetalk.examples.grpc.helloworld</groupId>
+  <artifactId>helloworld-maven</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>st-examples</name>
+  <url>https://servicetalk.io</url>
+
+  <properties>
+    <servicetalk.version>0.29.0</servicetalk.version>
```

Comment:
keeping versions up to date maybe painful down the line, but since this is meant to be an example from the user's perspective it is representative of actual use.

Reply:
Any way we can source `gradle.properties` and get the version?

Reply:
we maybe able to but because gradle and maven treat build dependencies differently using the version verbatim would lead to unresolved artifacts. I'll update the release script to bump this version.